assets/js/components/user-input/UserInputSelectOptions.js

```diff
@@ -77,11 +77,6 @@ export default function UserInputSelectOptions( {
 
 		if ( checkedEl ) {
 			focusOption( checkedEl );
-		} else {
-			const el = optionsRef.current.querySelector(
-				`input[type="${ optionType }"]`
-			);
-			focusOption( el );
```

Comment:
Let's not change this here. It's out of scope for the issue and I'm not sure if we should. Accessibility should probably be reviewed separately once the design iterations are implemented (or nearly done).

Reply:
Understood, reverted this change. Thank you for confirming, @aaemnnosttv!